ansible/roles/ocp4_workload_integreatly/tasks/remove_workload.yml

```diff
@@ -0,0 +1,12 @@
+---
+# Implement your Workload removal tasks here
+# ------------------------------------------
+
```

Comment:
No AWS resources of any kind were created that need cleanup?
S3?  DNS?  VIP?

Reply:
@newgoliath  It doesn't, no, the `ocp4_workload_integreatly` role only creates resources in the cluster